cmds/webboot/types.go

```diff
@@ -132,3 +132,13 @@ func (n *Network) Label() string {
 	}
 	return "Invalid wifi network."
 }
+
+// BackOption is a special menu Entry that says
+// the user wants to go back to the previous menu
+type BackOption struct{}
```

Comment:
Does it need backoption anymore? How about directly return the menu.BackRequest error? And the main function could recognize that user want to go back by this special error.

Reply:
Actually, in the next PR I hoping to move `BackOption` back to `types.go`. I had moved it to the menu package when I was changing how `PromptMenuEntry()` would handle an \<Esc> press, but now I think we're passing back an error type when the user presses \<Esc>.

Reply:
I see, but my think is how about remove all backoption struct, both in menu and in types. At the beginning ,we define backoption is because we want there is a entry in menu for user to go back and we want it's format is close to other entry.  But for now, since we plan to use <Esc> to go back and the menu do not need to display the back entry any more,  in my mind the backoption's usage is just letting the main function know that user want to go back, right? Then, why not use the menu.BackRequest error to do the same thing so that simplify the code?

Reply:
Yeah, actually that's a good idea. I was thinking of adding an error handler in the main loop at `webboot.go` where we can check the error types. That way we can handle back options, and also exit with 0 if the user pressed \<Ctrl+d>

Reply:
Yeah good idea!